src/state.rs

```diff
@@ -72,6 +76,14 @@ impl BarState {
             self.state.started = now;
         }
 
+        if let Reset::Formatters | Reset::All = mode {
```

Comment:
Let's not do a separate `Reset` variant for this, just reset the formatters for `All`.